src/app/(admin)/admin/_utils/common.js

```diff
@@ -23,6 +24,27 @@ export function transformTherapyPrices(therapies, therapyPrices) {
   );
 }
 
+export function isSpecifiedWorkTime(workTime) {
+  return workTime.some(day => day.isDayOff === false || day.isDayOff || day.time);
+}
+
+export function transformWorkTime(workTime) {
+  const specified = isSpecifiedWorkTime(workTime);
+  if (!specified) return [];
```

Comment:
You could omit creating extra-variable and simplify code like this:
```javascript
  if (!isSpecifiedWorkTime(workTime)) return [];
```

Reply:
Changed it